composer.json

```diff
@@ -9,16 +9,17 @@
     ],
     "authors": [
         {
-            "name": "Manny Isles",
-            "email": "manny@intergo.com.cy",
+            "name": "Intergo Telecom",
+            "email": "italosm@intergo.com.cy",
             "homepage": "https://intergo.com.cy",
             "role": "Developer"
         }
     ],
     "require": {
         "php": ">=7.1.3",
-        "guzzlehttp/guzzle": "~6.0",
-        "intergo/sms.to-php": "0.0.*"
+        "guzzlehttp/guzzle": "7.*",
+        "intergo/sms.to-php": "dev-add-sdk-for-all-endpoints",
```

Comment:
Set this to 1.0.0 as I merged it and pulled it on packagist.

Reply:
Done